ui/src/app/views/app/component.html

```diff
@@ -7,6 +7,7 @@
 <nav *ngIf="(user | async).isSignedIn" style="margin-top: 1rem;">
   <a routerLink="/login" routerLinkActive="active">Switch User</a>
   <a routerLink="/repository" routerLinkActive="active">Select CDR</a>
+  <a routerLink="/workspace" routerLinkActive="active">My Workspace</a>
```

Comment:
Eventually we'll need to get this information from the server. For now just use a couple of placeholders (which we can fhen tweak as match the workspace that @dmohs sets up for testing). If we do this right, the user can also change the values in the URL and actually (try to)  see a difference workspace :-) this could be helpful for demonstrations.